cdap-ui/app/cdap/components/AbstractWidget/AbstractMultiRowWidget/index.tsx

```diff
@@ -151,7 +151,7 @@ export default class AbstractMultiRowWidget<
     this.onChange();
   };
 
-  public constructValues = () => {
+  public constructValues = (): any => {
```

Comment:
Same here. We should try avoiding using `any` but if we need to we should provide a comment to justify the necessity.

Reply:
done